src/platform/forms-system/test/js/components/Telephone.unit.spec.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import Telephone, { CONTACTS } from '../../../src/js/components/Telephone';
+
+describe('Widget <Telephone />', () => {
+  // custom numbers
+  it('should render a number', () => {
+    const wrapper = shallow(<Telephone contact="8005551212" />);
+    const props = wrapper.props();
+    expect(props.href).to.equal('tel:+18005551212');
+    expect(props['aria-label']).to.equal('800. 5 5 5. 1 2 1 2');
+    expect(wrapper.text()).to.equal('800-555-1212');
+    wrapper.unmount();
+  });
+  it('should render a number with a leading "1"', () => {
+    const wrapper = shallow(<Telephone contact="1-800-555-1000" />);
+    const props = wrapper.props();
+    expect(props.href).to.equal('tel:+18005551000');
+    expect(props['aria-label']).to.equal('800. 5 5 5. 1000');
+    expect(wrapper.text()).to.equal('800-555-1000');
+    wrapper.unmount();
+  });
+  it('should throw an error when not passed a number', () => {
+    expect(() => {
+      const wrapper = shallow(<Telephone />);
+      wrapper.unmount();
+    }).to.throw('Telephone: "" does not match the pattern (###-###-####)');
+  });
+  it('should throw an error when number is less than 10-digits', () => {
+    expect(() => {
+      const wrapper = shallow(<Telephone contact="4321" />);
+      wrapper.unmount();
+    }).to.throw(`Telephone: "4321" does not match the pattern (###-###-####)`);
+  });
+  it('should throw an error when number is more than 10-digits', () => {
+    expect(() => {
+      const wrapper = shallow(<Telephone contact="01234567891" />);
+      wrapper.unmount();
+    }).to.throw(
+      'Telephone: "01234567891" does not match the pattern (###-###-####)',
+    );
+  });
+
+  // known numbers
+  it('should render a known number', () => {
+    const wrapper = shallow(<Telephone contact={CONTACTS.DS_LOGON} />);
```

Comment:
Love it!

Reply:
Hmm, maybe `platform/utilities/ui` because it's shorter? LOL

Reply:
I think that could work, but that directory seems like general-use functions versus a component...I'm not sure. We may want to migrate this soon over to [formation-react](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/tree/master/packages/formation-react/src/components) anyway so may not be a big deal anyway

Reply:
I'd also like to see this pattern added to the VA design system documentation, for how to handle phone numbers. I'll make sure this is added to the in-progress spreadsheet. 

Thanks so much, @Mottie , for getting this implemented!

Reply:
@jenstrickland we can definitely get this added onto https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/ because that's a dev site, but design.va.gov is more formal (there's a Design System Council) so it might be trickier

Reply:
But yeah, totally agree, because I'd love to see the plain HTML listed in https://design.va.gov/components/

Reply:
Coincidentally, I mentioned it earlier this morning to Ryan Thurwell when we worked on the specs deliverables for design system updates. I can write up something, and submit to the design council for review.